src/main/java/life/qbic/portal/portlet/MetadataProperty.java

```diff
@@ -0,0 +1,58 @@
+package life.qbic.portal.portlet;
+
+import java.util.Objects;
+
+/**
+ * This class describes a property of metadata for an image
+ * @param <T> The type of the contained value
+ *
+ */
+public class MetadataProperty<T> {
+  private final String name;
+  private final T value;
+  private final String description;
```

Comment:
Nice! You used `private final`!

Reply:
I'd like to take this moment to thank @KochTobi, who wrote this class for giving me the opportunity of hijacking this compliment!